packages/frontend-shared/cypress/support/mock-graphql/stubgql-Registry.ts

```diff
@@ -0,0 +1,29 @@
+import { stubWizard } from './stubgql-Wizard'
+import type { MaybeResolver } from './clientTestUtils'
+import { stubMutation } from './stubgql-Mutation'
+import { stubQuery } from './stubgql-Query'
+import { stubGlobalProject, stubProject } from './stubgql-Project'
+import { CloudOrganizationStubs, CloudProjectStubs, CloudRecordKeyStubs, CloudRunStubs, CloudUserStubs } from './stubgql-CloudTypes'
+import { stubMigration } from './stubgql-Migration'
+import type { CodegenTypeMap } from '../generated/test-graphql-types.gen'
+
+type MaybeResolveMap = {[K in keyof CodegenTypeMap]: MaybeResolver<CodegenTypeMap[K]>}
+
+export const GQLStubRegistry = {
```

Comment:
Moved this out to a separate file, so it's clear this is where to put the "registered" stubs for individual types for component testing